README.md

```diff
@@ -1,19 +1,44 @@
-# Git New
+# Generic New Project Template
 
-_A cloneable Git template for repeatable and consistent project bootstraps_
+_A Git template for bootstrapping a variety of projects with a consistent set of control files._
 
-# Table of Contents
+## Instructions
 
-- [Git New](#git-new)
-- [Table of Contents](#table-of-contents)
+1. Create your new repository by clicking the "Use this template" [button](https://github.com/tforster/git-new/generate) from the GitHub repository.
+1. Follow the GitHub prompts to complete the configuration of your new repository
+1. Clone your new repository to your local develop environment
+1. Install NPM dependencies including linters and code prettiers `npm i`
+1. Edit [README.md](README.md)
+   - Edit the title to match your project
+   - Edit the description to describe your project
+   - Edit Prerequisites, Setup and Configuration, Usage, For Users and Meta sections
+1. Edit [package.json](package.json)
+   - Update the title to match the title of this README
+   - Update the description to match the description of this README
+   - Edit the semantic version to match your project requirements
+1. Update your LICENSE
+   - Edit the year and fullname variables in [LICENSE.txt](LICENSE.txt)
+   - The provided LICENSE file implements the MIT license as per [https://choosealicense.com/licenses/mit](https://choosealicense.com/licenses/mit)
+   - For closed source delete the LICENSE.txt file and update the package.json license property to "UNLICENSED"
+   - If neither MIT or UNLICENSED suit your needs consider creating a LICENSE.txt from [https://choosealicense.com/](https://choosealicense.com/)
+1. Update [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md). For more information on code of conduct see [https://opensource.guide/code-of-conduct/](https://opensource.guide/code-of-conduct/)
+1. Update [CONTRIBUTING.md](CONTRIBUTING.md). For more information about open source contributions see [https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors)
+1. Delete any unwanted files in /docker (or the entire /docker folder if you're not containerising)
+1. Update [CHANGELOG.md](CHANGELOG.md)
+1. Delete this instruction block
```

Comment:
Wondering how much of this could be automated with a script that removes itself after it is complete?  Perhaps something I could contribute to this project.

Reply:
Answer: a lot :)